scripts/ci/thumbnails.py

```diff
@@ -6,15 +6,32 @@
 import argparse
 from io import BytesIO
 from pathlib import Path
-from typing import Generator
+from typing import Any, Dict, Generator
 
-import frontmatter
 import requests
+import tomlkit
```

Comment:
this should be added to some `requirements.txt`, and `frontmatter` should be removed from it